play/src/front/Streaming/BroadcastService.ts

```diff
@@ -3,16 +3,28 @@ import { libJitsiFactory } from "./Jitsi/LibJitsiFactory";
 import { JitsiConferenceWrapper } from "./Jitsi/JitsiConferenceWrapper";
 import { jitsiConferencesStore } from "./Jitsi/JitsiConferencesStore";
 import { megaphoneEnabledStore } from "../Stores/MegaphoneStore";
-import { get, Readable, Unsubscriber } from "svelte/store";
+import { derived, get, Readable, Unsubscriber } from "svelte/store";
 import { ForwardableStore } from "@workadventure/store-utils";
 import { JitsiTrackWrapper } from "./Jitsi/JitsiTrackWrapper";
+import JitsiConnection from "lib-jitsi-meet/types/hand-crafted/JitsiConnection";
+import { Space } from "../Space/Space";
+import pLimit from "p-limit";
+
+class BroadcastSpace extends Space {
+    public jitsiConference: JitsiConferenceWrapper | undefined;
+    public unsubscribes: Unsubscriber[] = [];
```

Comment:
You should subscribe and unsubscribe in the same class. Either move the unsubscribe to BroadcastService or move the subscribe to BroadcastSpace